### DISCUSSION TITLE
"svgPoints()" doesn't exist ..?

### DISCUSSION THREAD

**Discussion by 3Dietrich** [dinther/pathbuilder#6] [Q&A] [answered]
in readme: 

svgPoints | svgPoints(path_string)
-- | --

seems to be wrong/outdated. I guess you want to write 
svgShape | svgShape(path_string)
--|--

**Comment by dinther** [comment, marked as answer]
Hi, thanks for the feedback on the missing svgPoints function. I just made another release V0.81 I hope this releases fixes it. I come back to your other points tomorrow. It's getting late here down-under. Cheers.

By the way, the arc function isn't implemented yet just so you know.
